Gruntfile.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(grunt) {
   grunt.initConfig({
     _test_runner: '_mocha',
     _istanbul: 'istanbul cover --dir',
-    _unit_args: '--recursive -t 10000 ./test/',
+    _unit_args: '--recursive -t 10000 ./test/test_request-id.js',
```

Comment:
Is this a change you made locally to run a single file? Doesn't look like it should be committed.